geojson_pydantic/geometries.py

```diff
@@ -27,6 +28,20 @@ class _GeometryBase(BaseModel, abc.ABC):
     def __geo_interface__(self):
         return self.dict()
 
+    @classmethod
+    def validate(cls, value):
+        try:
+            return cls(**json.loads(value))
+        except Exception:
```

Comment:
Is the goal of this except block to catch errors on `json.loads`?  If so I think the except block should be more narrowly scoped to `TypeError` instead of `Exception`.  As written we risk swallowing validation errors if JSON is parseable by `json.loads`.

Reply:
> As written we risk swallowing validation errors if JSON is parseable by json.loads

Oh yes, you are right, I will update this to fix this issue.